apache-geode/src/main/java/com/baeldung/geode/functions/CustomerWithMaxAge.java

```diff
@@ -0,0 +1,39 @@
+package com.baeldung.geode.functions;
+
+import com.baeldung.geode.Customer;
+import org.apache.geode.cache.Region;
+import org.apache.geode.cache.execute.Function;
+import org.apache.geode.cache.execute.FunctionContext;
+import org.apache.geode.cache.execute.RegionFunctionContext;
+
+import java.util.Comparator;
+import java.util.Map;
+import java.util.Optional;
+
+public class CustomerWithMaxAge implements Function<Customer> {
```

Comment:
Is this class still necessary? If not, let's remove it.

Reply:
Ok Sure ...